src/stream/src/common/log_store/in_mem.rs

```diff
@@ -274,17 +274,19 @@ mod tests {
         let epoch2 = init_epoch + 2;
 
         let ops = vec![Op::Insert, Op::Delete, Op::UpdateInsert, Op::UpdateDelete];
-        let mut builder = DataChunkBuilder::new(vec![DataType::Int64, DataType::Varchar], 10000);
-        for i in 0..ops.len() {
+        let mut builder = StreamChunkBuilder::new(10000, vec![DataType::Int64, DataType::Varchar]);
+        for (i, op) in ops.into_iter().enumerate() {
             assert!(builder
-                .append_one_row(OwnedRow::new(vec![
-                    Some(ScalarImpl::Int64(i as i64)),
-                    Some(ScalarImpl::Utf8(format!("name_{}", i).into_boxed_str()))
-                ]))
+                .append_row(
+                    op,
+                    OwnedRow::new(vec![
+                        Some(ScalarImpl::Int64(i as i64)),
+                        Some(ScalarImpl::Utf8(format!("name_{}", i).into_boxed_str()))
+                    ])
+                )
```

Comment:
Looks like we can `impl Row for [Datum; N]` so that malloc in `OwnedRow` can be avoided here. (although it's a test and performance is not critical here)

Reply:
Yeah, we already have this -> https://github.com/risingwavelabs/risingwave/blob/abea7f502d8f4c43e49f9a081eab53c4b5986b4f/src/common/src/row/mod.rs#L302-L304